mintpy/utils/plot.py

```diff
@@ -1273,14 +1273,14 @@ def plot_gps(ax, SNWE, inps, metadata=dict(), print_msg=True):
                                                     ref_site=inps.ref_gps_site,
                                                     gps_comp=inps.gps_component)[1] * unit_fac
                 gps_data = dis[-1] - dis[0]
-                
+
             if np.isnan(gps_data) == False:
                 listGPS.append([site_names[i],site_lons[i], site_lats[i], gps_data])
             else: pass
 
             # save calculated GPS velocities to CSV file
             import csv
-            csv_columns = ['SiteID','Lon','Lat', 'Vel ('+str(inps.disp_unit)+' in LOS)']
+            csv_columns = ['Site ID','Lon','Lat', 'Vel ('+str(inps.disp_unit)+' in LOS)']
```

Comment:
Hi @ehavazli, are you sure about this change? In a previous commit (https://github.com/insarlab/MintPy/pull/243), you removed the whitespace.

Reply:
it shouldn't be there. Let me close this one and fix the fork. I will make the changes to bootSrap.py naming comvention. That should be the only commit.